README.md

```diff
@@ -2,6 +2,7 @@
 
 Dink sends webhook messages upon noteworthy in-game events.
 While Dink supports the Discord webhook format (with rich embeds and optional screenshots), it also includes additional metadata that allows custom webhook servers to analyze messages or even generate their own messages.
+Examples of the additional metadata can be found [here](https://github.com/Felanbird/DinkPlugin/blob/chore/move-nerd-data-out-of-base-readme/docs/json-examples.md).
```

Comment:
if we convert this to a relative path, would the hub page still point to the correct link?

Reply:
I don't know

Reply:
relative links seem to work fine
https://runelite.net/plugin-hub/show/tombs-of-amascut
https://raw.githubusercontent.com/LlemonDuck/tombs-of-amascut/master/README.md